docs/references/general/grid.md

```diff
@@ -0,0 +1,103 @@
+<!-- Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan 
+author(s): Marina Schmidt 
+
+Disclaimer: License under GNU v3.0, the file is found in the root directory under LICENSE 
+
+-->
+# GRID files 
+
+GRID files are post-processed data product from FITACF files.
+
+## Naming Conventions
+
+### Single GRID files
+
+Currently the common naming convention for a GRID file is:
+
+> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.grid
+
+SuperDARN radars routinely change operating frequency. Some operating modes use multiple frequencies, either at the same time, or by alternating between frequencies. As a result, some files from some radars include all records for all frequencies in the same file and other radars separate out the individual frequencies into separate files, designated by a channel letter in the file name:
```

Comment:
This paragraph implies that a channel letter would be used to separate different operating frequencies that are not run on different radar channels.  Is that really true?  If not, the wording needs to be altered.

Reply:
@aburrell  I wonder if this should be documented more centrally, since it applies not just to grid/map files but to the rawacf and fitacf files too.

Reply:
> I wonder if this should be documented more centrally

I think that would be a good thing to do.  I also think we should be very careful not to accidentally create confusion by conflating frequency bands and channels at all levels of documentation.

Reply:
@aburrell I don't think this is always the case - I think the CPID would change, but not necessarily the channel letter. Perhaps @egthomas knows more on this.

Reply:
I agree with all of the above comments by @aburrell, @ecbland, and @mtwalach - the separation of control programs, channels, and frequencies can be complicated.  In this case, I am not aware of a control program which creates a file for a separate channel solely because of a change in frequency (although that does not mean such a control program does not exist, perhaps Borealis is doing so?); certainly there have been Stereo modes which use a different frequency on the second channel, but that information is usually contained in the same rawacf / fitacf file and identified via the channel field.

Reply:
If anyone wants to write something here please do so :) I am mainly just starting the draft.

Reply:
I think this is getting beyond the scope of documenting grid files. Maybe just keep it simple here, and then provide more explanation on the file naming elsewhere (when DSWG has standardised it)?

```suggestion
Some radars provide separate data files for each channel. In this case, the channel is specified after the 3-letter station ID:
```

Reply:
I think this is good.  It doesn't explain how channels work (which is FINE) and it doesn't imply they operate a specific way.